pymc3/__init__.py

```diff
@@ -27,12 +27,6 @@
         handler = logging.StreamHandler()
         _log.addHandler(handler)
 
-# Set start method to forkserver for MacOS to enable multiprocessing
-# Closes issue https://github.com/pymc-devs/pymc3/issues/3849
-sys = platform.system()
-if sys == "Darwin":
-    new_context = mp.get_context("forkserver")
-
```

Comment:
Why was this dropped? This settled a pretty nasty bug on MacOs. Do we handle this otherwise now?

Reply:
Oh, this was supposed to end up in a different PR.
I seriously doubt that this solves any issue though, because the context isn't used anywhere. This will be trivial to fix after https://github.com/pymc-devs/pymc3/pull/3991 though.

Reply:
I don't have your knowledge on these issues, but if I understood the python doc correctly, getting the context was a safer way to change it than to set it explicitly (quite counterintuitive) and I'm pretty confident that solved the bug I and a bunch of users were having and that manifested on the terminal when running MPL in NBs ([see the PR](https://github.com/pymc-devs/pymc3/pull/3919) for more context).
In any case, if there is a better way to handle this problem now, I think it is indeed better to explicitely drop this bit in a dedicated PR than hidden in this one -- do you agree?
Also, how is #3991 related to MacOS?

Reply:
You are completely right, this doesn't belong in this PR, I took it out again.

The problem is that just asking for the context doesn't do anything. It only changes something, if you don't use the objects from mulitprocessing, but from the context itself. But since it is never used, it should have absolutely no effect. If you look at the source of multiprocessing, you can see that get_context just returns an object that already exists in a dict: https://github.com/python/cpython/blob/master/Lib/multiprocessing/context.py#L187

The idea of those contexts is that you can use those instead of the multiprocessing namespace to get different behaviour. So you can create a new pool by using `ctx.Pool` instead of `multiprocessing.Pool`.

#3991 makes the context configurable, so that we don't use multiprocessing directly but through a context.

Reply:
> You are completely right, this doesn't belong in this PR, I took it out again.

Great, thanks, gonna review then. If approved, is it ready for merge now (note that we're [adding py36 again](https://github.com/pymc-devs/pymc3/pull/4003) in the end 😬).

> The problem is that just asking for the context doesn't do anything.

I agree with that actually 😅 I was confused when playing with this in a NB at the time -- since it had an effect on the bug I was referring to (or at least _seemed_ to have an effect), I went with this 🤷‍♂️ But all you're saying makes sense.

> #3991 makes the context configurable, so that we don't use multiprocessing directly but through a context.

I may have questions about that but I'll ask them directly on the PR 😉